selectel/resource_selectel_mks_cluster_v1.go

```diff
@@ -0,0 +1,560 @@
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"count": {
+							Type:     schema.TypeInt,
+							Required: true,
+						},
+						"keypair_name": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"affinity_policy": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"cpus": {
+							Type:     schema.TypeInt,
+							Optional: true,
+						},
+						"ram_mb": {
+							Type:     schema.TypeInt,
+							Optional: true,
+						},
+						"volume_gb": {
+							Type:     schema.TypeInt,
+							Optional: true,
+							Computed: true,
+						},
+						"volume_type": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
+						},
+						"local_volume": {
+							Type:     schema.TypeBool,
+							Optional: true,
+							Computed: true,
+						},
+						"flavor_id": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
+						},
+						"cluster_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"created_at": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"updated_at": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"nodes": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"id": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+									"created_at": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+									"updated_at": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+									"hostname": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+									"ip": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+									"nodegroup_id": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			"enable_autorepair": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+				ForceNew: false,
+			},
+			"enable_patch_version_auto_upgrade": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+				ForceNew: false,
+			},
+			"network_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"subnet_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"maintenance_window_start": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: false,
+			},
+			"maintenance_window_end": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"maintenance_last_start": {
```

Comment:
I think we can remove this since this value won't show the actual latest timestamp.
It will change every 24h in MKS API but will remain unchanged in the local Terraform state.